src/AVFoundation/Enums.cs

```diff
@@ -655,4 +663,28 @@ public enum AVMovieWritingOptions : nuint
 		TruncateDestinationToMovieHeaderOnly = (1 << 0)
 	}
 #endif
+
+	[NoTV, NoWatch, NoMac, iOS (10,0)]
+	[Native]
+	public enum AVCaptureColorSpace : nint
+	{
+		Srgb = 0,
+		P3D65 = 1
+	}
+
+	[iOS (8,0)]
+	[Native]
+	public enum AVMusicTrackLoopCount : nint
+	{
+		Forever = -1
+	}
+
+	[NoWatch, iOS (10,0), TV (10,0), Mac (10,12)]
+	[Native]
+	public enum AVPlayerTimeControlStatus : nint
+	{
+		Paused,
+		WaitingToPlayAtSpecifiedRate,
+		Playing
+	}
```

Comment:
All this should be in the previous one, I did a rebase, it should not show in the diff AFAIK after a rebase.